Microsoft.Toolkit.Parsers/Rss/RssHelper.cs

```diff
@@ -140,6 +140,36 @@ public static string GetSafeElementString(this XElement item, string elementName
             return GetSafeElementString(item, elementName, item.GetDefaultNamespace());
         }
 
+        /// <summary>
+        /// Get item string values for xelement and element name.
+        /// </summary>
+        /// <param name="item">XElement item.</param>
+        /// <param name="elementName">Name of the element.</param>
+        /// <returns>Safe list of string values.</returns>
+        public static IEnumerable<string> GetSafeElementsString(this XElement item, string elementName)
+        {
+            return GetSafeElementsString(item, elementName, item.GetDefaultNamespace());
```

Comment:
Could this throw an exception if item == null?

Reply:
In the called method item is verified whether it's null.

Reply:
Could this method  be called outside of this class since it's public (internal public, but still public)?

Reply:
As far as I can tell, I have added these two methods that are basically similar to `GetSafeElementString`, so I really couldn't tell, but i believe they will have basically the same impact as the existing ones.

Reply:
Good point.